firebase/utils/blogUtils.js

```diff
@@ -0,0 +1,22 @@
+import { collection, doc, getDoc, serverTimestamp, setDoc } from "firebase/firestore";
+
+import { firestore } from "@/firebase/firebase";
+
+export const blogCollection = collection(firestore, "blog");
+
+export const createBlog = async (uid, data) => {
+  const blogRef = doc(blogCollection);
+  await setDoc(blogRef, { ...data, userId: uid, createdAt: serverTimestamp() });
+};
+
+export const updateBlog = async (uid, data) => {
+  const blogRef = doc(blogCollection, uid);
+  await setDoc(blogRef, data, { merge: true });
+};
+
+export const isBlogExists = async (uid) => {
```

Comment:
reword, maybe `isExistingBlog` or `doesBlogExist`

Reply:
// check if blog exists by id
export const isExistingBlog = async (id) => {
  const blogRef = doc(blogCollection, id);
  const blog = await getDoc(blogRef);

  return blog.exists();
};


function was renamed to 'isExistingBlog'